src/result_processor.c

```diff
@@ -891,6 +894,13 @@ static int rpSafeLoaderNext_Accumulate(ResultProcessor *rp, SearchResult *res) {
   // Done loading. Unlock Redis
   RedisModule_ThreadSafeContextUnlock(sctx->redisCtx);
 
+  if (isQueryProfile) {
+    clock_gettime(CLOCK_MONOTONIC, &rpEndTime);
+    rs_timersub(&rpEndTime, &rpStartTime, &rpEndTime);
+    rs_timeradd(&rpEndTime, &rp->GILTime, &rp->GILTime);
+    rs_timeradd(&rpEndTime, &rp->parent->GILTime, &rp->parent->GILTime);
```

Comment:
Why do we need both increments? Seems redundant..
Are both printed\used? What's the difference between them?